ssh/tailssh/tailssh.go

```diff
@@ -375,7 +376,12 @@ func (srv *server) handleAcceptedSSH(ctx context.Context, s ssh.Session, ci *ssh
 		stdin.Close()
 	}()
 	go func() {
-		_, err := io.Copy(s, stdout)
+		// Write to s.Channel directly, avoiding gliderlab/ssh's (*session).Write
+		// call that translates newline endings, which we don't need.
+		// See https://github.com/tailscale/tailscale/issues/4146.
+		// TODO(bradfitz,maisem): remove this reflect hackery once gliderlab/ssh changes
+		// are all in.
+		_, err := io.Copy(reflect.ValueOf(s).Elem().FieldByName("Channel").Interface().(io.Writer), stdout)
```

Comment:
nit: can you do this in a different line?

Reply:
done, with more comments